packages/experiments/src/components/VerticalPersona/VerticalPersona.view.tsx

```diff
@@ -0,0 +1,30 @@
+import * as React from 'react';
+import { PersonaCoin } from '../PersonaCoin';
+import { Text } from '../Text';
+import { IVerticalPersonaComponent } from './VerticalPersona.types';
+
+export const VerticalPersonaView: IVerticalPersonaComponent['view'] = props => {
+  const { root, text, secondaryText } = props.classNames;
+  const coinProps = props.coinProps || {};
+
+  return (
+    <div className={root}>
+      <PersonaCoin
+        text={props.text}
+        size={coinProps.size}
+        initials={coinProps.initials}
+        imageUrl={coinProps.imageUrl}
+        coinColor={coinProps.coinColor}
+        initialsColor={coinProps.initialsColor}
+        presence={coinProps.presence}
+        onPhotoLoadingStateChange={coinProps.onPhotoLoadingStateChange}
+      />
+      <Text wrap className={text}>
+        {props.text}
+      </Text>
+      <Text wrap className={secondaryText}>
+        {props.secondaryText}
+      </Text>
```

Comment:
This looks like it could be a good candidate for Slots similar to button (#7308). Slot candidates here could be root, coin, and text fields and would automatically apply classNames and and provide full slot flexibility..

Reply:
Thanks, I'll need to look into this a bit as I'm not yet sure what "Slots" means (yet)

Reply:
#7012 has a preliminary documentation writeup. 😄